resources/views/actions/edit.blade.php

```diff
@@ -70,22 +70,54 @@
                     <div class="form-item">
                         <label class="field-label">Post Details</label>
                         <label class="option -checkbox">
-                          <input type="checkbox" id=reportback name=reportback value=1 {{ old('reportback') || $action->reportback ? 'checked="checked"' : '' }}>
+                          <input type="checkbox" id=reportback name=reportback value=1
+                            @if ($errors->any()))
+                                @if (old('reportback'))
+                                    checked="checked"
+                                @endif
+                            @elseif ($action->reportback)
+                                checked="checked"
+                            @endif
+                          >
```

Comment:
yay! so glad this is working! would it make sense to pull this logic into a helper? can we really do that with blade files? 🤔

Reply:
I was looking online and [this could be a solution](https://stackoverflow.com/questions/47879154/laravel-checkbox-state-after-failed-validation)? Would this make sense to make into a separate PR since this one is a little hefty?

Reply:
that seems like it could work! totally up to you!

Reply:
@katiecrane I think that I will pull that into another PR since this one is getting a little beefy! Made a card for that [here](https://www.pivotaltracker.com/n/projects/2019429/stories/164772481)!